swagger/routes/destination.yaml

```diff
@@ -0,0 +1,141 @@
+/{version}/destinations/{destination}:
+  post:
+    tags:
+      - Destination
+    summary: Destination processor transformation
+    parameters:
+      - in: path
+        name: version
+        description: The version of the destination
+        required: true
+        schema:
+          type: string
+      - in: path
+        name: destination
+        description: The destination identifier
+        required: true
+        schema:
+          type: string
+    requestBody:
+      description: Request Body
+      required: true
+      content:
+        application/json:
+          schema:
+            type: array
+            items:
+              $ref: '../components/schemas/processorTransformationRequest.yaml'
+    responses:
+      200:
+        description: Success
+        content:
+          application/json:
+            schema:
+              $ref: '../components/schemas/processorTransformationResponse.yaml'
+      400:
+        description: Failure
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                metadata:
+                  $ref: '../components/schemas/metadata.yaml'
+                statusCode:
+                  type: number
+                error:
+                  type: string
+                statTags:
+                  type: object
+
+/routerTransform:
+  post:
+    tags:
+      - Destination
+    summary: Destination router transformation
+    requestBody:
+      description: Request Body
+      required: true
+      content:
+        application/json:
+          schema:
+            $ref: '../components/schemas/routerTransformationRequest.yaml'
+    responses:
+      200:
+        description: Success
+        content:
+          application/json:
+            schema:
+              $ref: '../components/schemas/routerTransformationResponse.yaml'
+      400:
+        description: Failure
```

Comment:
I dont think this is the correct representation of success response can you please check

Reply:
updated.